packages/core/core/src/services/document-service/repository.ts

```diff
@@ -184,7 +184,7 @@ export const createContentTypeRepository: RepositoryFactoryMethod = (
       entriesToClone,
       async.pipe(
         validateParams,
-        omit('id'),
+        omit(['id', 'createdAt']),
```

Comment:
might as well include the `updatedAt` value!

```suggestion
        omit(['id', 'createdAt', 'updatedAt']),
```